generators/PHPythia8/PHPythia8.C

```diff
@@ -41,17 +40,21 @@ PHPythia8::PHPythia8(const std::string &name)
   , _pythiaToHepMC(NULL)
   , _save_integrated_luminosity(true)
   , _integral_node(nullptr)
+  , _legacy_vertexgenerator(false)
 {
   char *charPath = getenv("PYTHIA8");
   if (!charPath)
-  {
-    cout << "PHPythia8::Could not find $PYTHIA8 path!" << endl;
-    return;
-  }
+    {
+      cout << "PHPythia8::Could not find $PYTHIA8 path!" << endl;
+      return;
+    }
 
   std::string thePath(charPath);
   thePath += "/xmldoc/";
   _pythia = new Pythia8::Pythia(thePath.c_str());
```

Comment:
Is `new` here necessary?  It seems to me that `_pythia` will point to `ppGen` or `pnGen` later.

Reply:
Yes, I have deleted L54 as our discussion.